packages/native/src/components/MapView/MapCallout.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { TouchableOpacity } from "react-native";
+import { InfoWindow } from "@react-google-maps/api";
+import { markerContext } from "./MapMarker";
+import { CalloutProps } from "./types";
+
+const MapCallout: React.FC<CalloutProps> = ({
+  anchor,
+  onPress = () => {},
+  children,
+}) => {
+  const { toggleCallout, position, calloutOpened } =
+    React.useContext(markerContext);
+  const handleClose = () => toggleCallout(false);
+  const handlePress = () => onPress();
+  return calloutOpened ? (
+    <TouchableOpacity onPress={handlePress}>
```

Comment:
This seems like an unnecessary abstraction to me
```suggestion

  return calloutOpened ? (
    <TouchableOpacity onPress={onPress}>
```